tests/ignite/metrics/test_epoch_metric.py

```diff
@@ -103,7 +103,7 @@ def compute_fn(y_preds, y_targets):
     targets = torch.cat([output1[1], output2[1], output3[1]], dim=0)
 
     result = em.compute()
-    assert result == compute_fn(preds, targets)
+    assert result == pytest.approx(compute_fn(preds, targets), rel=1e-6)
```

Comment:
This change is surprising, why do you need this?

Reply:
because it was not exactly the same and the comparison failed

Reply:
On MPS ?

Reply:
Make a separate PR for this metric